server.js

```diff
@@ -2,6 +2,10 @@ const app = require('./app')
 
 app.set('port', process.env.PORT || 3001);
 
-app.listen(app.get('port'), () => console.log(`App is running on localhost: ${app.get('port')}`));
+app.listen(app.get('port'), () => {
+  console.log(`App is running on localhost:${app.get('port')}.`);
+});
+
```

Comment:
I had this on one line but I think it's more legible the way you wrote it.  Thanks!